server/routes/transport.js

```diff
@@ -7,7 +7,7 @@ const routeAccessibility = (req, res, next) => {
   if (authCookie) {
     const returnCheckAuth = service.checkAuthStatus(authCookie);
     if (returnCheckAuth === '403') {
-      return res.sendStatus(403);
```

Comment:
if you're removing this return then on line 11 you should have an `else if` statement